core/examples/luigi-example-next/pages/home.js

```diff
@@ -0,0 +1,24 @@
+import { useEffect } from 'react';
+import Head from 'next/head';
+
+export default function Home() {
+  useEffect(() => {
+    require('@luigi-project/client/luigi-client.js');
+  }, []);
+
+  return (
+    <>
+      <Head>
+        <link
+          href="//unpkg.com/fundamental-styles@latest/dist/fundamental-styles.css"
```

Comment:
I am not getting any fundamental styles applied to the micro-frontends. I noticed that you switched from Global CSS (in _app.js) to internally linked to the `<Head>` inside each functional component. None of the options showed any fundamental styles applied for me. Let me know if that's the case for you as well or not 🤔, since others also didn't seem to have this behaviour. 
  
This probably gets to be on a lower priority, since we don't want to be spending too much time on it, and it's a Next.js specific thing so feel free to mark this as resolved anyway 👍 .

Reply:
I cannot reproduce this issue, the fundamental styles applied to micro-frontends successfully on my side.